docs/2_2_common_mechanisms.adoc

```diff
@@ -950,14 +951,14 @@ with the following meanings:
 
 [[fmi3IntervalNotYetKnown,`fmi3IntervalNotYetKnown`]]
```

Comment:
Now I see why you defined t_event as the point in time when GetInterval is called. This qualifier value will really make it diffucult for the importer to react on the case where the callback is called but the interval is not set..... Why did we define the countdown clock to be so complicated?!

Reply:
As always, the answer is: "because of Modelica!"
In general, the countdown interval might not just change (hence: call every time), but might also not be known at first. We need to account for this.